File: Arrays/linearsearch.js
let arr = [10, 20, 50, 60, 90];
let key = 90;
// for (let i = 0; i < arr.length; i++) {
// 	if (arr[i] === key) {
// 		console.log(`element found at ${i}`);
// 	} else if (arr[i] !== key) {
// 		continue;
// 	} else {
// 		console.log("element not found");
// 	}
// }
let index = arr.findIndex((element) => element === key);
if (index !== -1) {
	console.log(`element found at ${index}`);
} else {
	console.log("element not found");
}

File: Arrays/linearsearch.cpp
// linear search
#include <iostream>
using namespace std;
int main()
{
    int arr[5] = {20, 88, 99, 67, 97};
    int size = 5;
    bool flag = 0;
    int target = 6;
    for (int i = 0; i < size; i++)
    {
        if (arr[i] == target)
        {
            flag = 1
        }
    }
    if (flag == 1)
    {
        cout << "elemet found";
    }
    else
    {
        cout << "element not found";
    }

    return 0;
}